crates/polars-io/src/csv/read_impl/mod.rs

```diff
@@ -576,6 +576,17 @@ impl<'a> CoreReader<'a> {
             return Ok(df);
         }
 
+        if !self.ignore_errors {
```

Comment:
I don't want this. This does a whole scan of the file.

The csv-parser is designed to be fast for correct files. This isn't a csv-file sanitizer.